core/builtins/native/kotlin/Primitives.kt

```diff
@@ -25,65 +25,146 @@ package kotlin
 public class Byte private () : Number, Comparable<Byte> {
     default object {}
 
+/**
+ * Compares this value with the specified value for order.
+ * Returns zero if this value is equal to the specified other value, a negative number if its less than other, 
+ * or a positive number if its greater than other.
+ */
     public override fun compareTo(other: Byte): Int
+/**
+ * Compares this value with the specified value for order.
+ * Returns zero if this value is equal to the specified other value, a negative number if its less than other, 
+ * or a positive number if its greater than other.
+ */
     public fun compareTo(other: Char): Int
```

Comment:
Comparing byte with char for ordering probably requires more description about what it really means.